src/components/navbar/Navbar.js

```diff
@@ -3,16 +3,26 @@ import { translate } from 'react-i18next';
 import 'i18n/i18n';
 import styles from './navbar.scss';
 
+const handleClickOnNavigation = (id) => {
+  return (event) => {
+    event.preventDefault();
+    document.getElementById(id).scrollIntoView({
+      behavior: 'smooth',
+    });
+    window.history.pushState({}, '', `/#${id}`);
```

Comment:
There is an issue created to don't upadte the url so we should use buttons instead of anchors. Maybe we could try to do both in this PR

Reply:
I think that issue is only for the modal window links. I think is ok to set the hashes when going to these sections.

Reply:
@tureey I created that issue for modals. I added the id to the url to make the bookmarking possible.